### PR TITLE
Possible bug fixed + Minor refact

### PR DESCRIPTION
**Pull request by ivandrofly**

### CODE REVIEW

File: src/Logic/Utilities.cs [hunk 1]
```diff
@@ -3211,7 +3210,7 @@ private static string RemoveSpaceBeforeAfterTag(string text, string openTag)
                     break;
             }
 
-            if (closeTag.Length == 0 && openTag.Contains("<font ", StringComparison.Ordinal))
+            if (closeTag.Length == 0 && openTag.Contains("<font", StringComparison.Ordinal))
```
Comment (ivandrofly) [inline src/Logic/Utilities.cs]: Could be ```<font>```

File: src/Logic/Utilities.cs [hunk 2]
```diff
@@ -3185,18 +3185,17 @@ private static string RemoveSpaceBeforeAfterTag(string text, string openTag)
                 .Replace("</U>", "</u>")
                 .Replace("</FONT>", "</font>");
 
-            if (text.Contains("<FONT", StringComparison.Ordinal))
```
Comment (ivandrofly) [inline src/Logic/Utilities.cs]: IndexOf is already checking

File: src/Forms/DurationsBridgeGaps.cs [hunk 3]
```diff
@@ -145,6 +145,5 @@ private void DurationsBridgeGaps_KeyDown(object sender, KeyEventArgs e)
             if (e.KeyCode == Keys.Escape)
                 DialogResult = DialogResult.Cancel;
         }
-
```
Comment (niksedk) [inline src/Forms/DurationsBridgeGaps.cs]: I like a space here... makes it easy to add new methods...

Comment (ivandrofly) [inline src/Forms/DurationsBridgeGaps.cs]: :+1:

**Pull request closed without merging**